Library/Homebrew/cmd/config.rb

```diff
@@ -148,16 +148,14 @@ def describe_system_ruby
   end
 
   def describe_java
-    # java_home doesn't exist on all OS Xs; it might be missing on older versions.
-    return "N/A" unless File.executable? "/usr/libexec/java_home"
-
-    java_xml = Utils.popen_read("/usr/libexec/java_home", "--xml", "--failfast")
-    return "N/A" unless $?.success?
-    javas = []
-    REXML::XPath.each(REXML::Document.new(java_xml), "//key[text()='JVMVersion']/following-sibling::string") do |item|
-      javas << item.text
-    end
-    javas.uniq.join(", ")
+    # use $JAVA_HOME to find java version.
+    java_home = %x( echo $JAVA_HOME )
+    java_home.slice! "\n"
+    return "N/A" unless !java_home.empty?
```

Comment:
In fact, since `describe_java` returns a list of all the installed Java versions, it should probably do both `$JAVA_HOME` and `/usr/libexec/java_home` and combine their results (removing duplicates).